polynote-server/src/main/scala/polynote/NotebookRunner.scala

```diff
@@ -90,8 +90,8 @@ object NotebookRunner {
   } yield kernel
 
   private def shouldRunCell(cell: NotebookCell) = cell.language.toString match {
-    case "text" | "vega" | "markdown" => false
-    case _ => true
+    case "python" | "scala" | "sql" => true
```

Comment:
The problem with this is that hypothetically, plugins could add runnable languages. Plugins can't (currently) add non-runnable languages, so that's why that set is hard-coded.

We could check against the actual list of runnable languages (could be pulled from `Interpreter.Factories` environment).  That way when we actually do add a Scala-side interpreter for viz (hope to do that eventually, so we can get plots rendered by the notebook runner) it will just... start working.

Reply:
Okay. What you are saying really makes sense to me. Let me change the code in accordance with the `Interpreter.Factories` approach and quickly test it.

Reply:
Done, looks working. Check it out